build.gradle

```diff
@@ -48,11 +48,13 @@ allprojects {
   task functional(type: Test, description: 'Runs the functional tests.', group: 'Verification') {
     testClassesDirs = sourceSets.functionalTest.output.classesDirs
     classpath = sourceSets.functionalTest.runtimeClasspath
+    environment("APPINSIGHTS_INSTRUMENTATIONKEY", "test-key")
```

Comment:
you shouldn't need this its set in a properties file. if this was required the tests would be failing on mastetr

Reply:
OK. I followed the CMC config for this though.

Reply:
legacy, they would have missed it when they updated their java-logging version

Reply:
Do I remove it for integration & smoke test as well?

Reply:
should be safe.

Reply:
```suggestion
```